critiquebrainz/frontend/views/moderators.py

```diff
@@ -0,0 +1,20 @@
+from flask import Blueprint, render_template
+from flask import current_app
+from critiquebrainz.db import users as db_users
+
+moderators_bp = Blueprint('moderators', __name__)
+
+
+@moderators_bp.route('/')
+def moderators():
+    admins = current_app.config['ADMINS']
+    db_admins = db_users.get_many_by_mb_username(admins)
+    user_names = [item['display_name'] for item in db_admins]
```

Comment:
It would be easier to read this function if the same thing is named consistently. In this case `admin` and `user` mean the same.